src/adapter/bun/ssg.ts

```diff
@@ -1,28 +1,28 @@
 // @denoify-ignore
 /* eslint-disable @typescript-eslint/ban-ts-comment */
-import { mkdir, writeFile } from 'fs/promises'
 import { toSSG as baseToSSG } from '../../helper/ssg'
 import type { FileSystemModule, ToSSGAdaptorInterface } from '../../helper/ssg'
 
+// @ts-ignore
+const { write } = Bun
```

Comment:
The import is the same as for the Bun.file because it took a long time to break other parts of the Bun type when it was introduced.